packages/package-bundler/tsconfig.base.json

```diff
@@ -9,6 +9,7 @@
     "jsx": "react",
     "noImplicitAny": true,
     "noImplicitReturns": true,
-    "strict": true
+    "strict": true,
+    "noEmit": true
```

Comment:
How come this needs to be declared at the same time as the flag?

Reply:
The bundler is consumed by others, so we want to ensure it has the flag by default :)
Our flags are a coincidence per se, but put in to ensure that linting doesn't generate any files 😬

Reply:
Oh wrong place to comment this, pretty sure I saw this in our other packages, ah well no worries